exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -136,44 +114,44 @@ private string CurrentCategory()
             return "Rock";
         }
 
-        public bool WasCorrectlyAnswered()
+        public bool IsCorrectAnswer()
         {
             if (inPenaltyBox[currentPlayer])
             {
                 if (isGettingOutOfPenaltyBox)
                 {
                     Console.WriteLine("Answer was correct!!!!");
                     playersPurses[currentPlayer]++;
-                    Console.WriteLine(listOfAllPlayers[currentPlayer]
+                    Console.WriteLine(player.listOfAllPlayers[currentPlayer]
                             + " now has "
                             + playersPurses[currentPlayer]
                             + " Gold Coins.");
 
                     bool winner = HasPlayerWon();
                     currentPlayer++;
-                    if (currentPlayer == listOfAllPlayers.Count) currentPlayer = 0;
+                    if (currentPlayer == player.listOfAllPlayers.Count) currentPlayer = 0;
```

Comment:
When reading this line by itself it's not descriptive enough to full understand what is happening. You need something like a SetPlayerToPositionZero() method.